src/extension.ts

```diff
@@ -12,10 +12,24 @@ function activate ( context: vscode.ExtensionContext ) {
   Utils.state = context.globalState;
   Utils.initCommands ( context );
 
+  context.subscriptions.push ( vscode.window.onDidChangeWindowState ( onDidChangeWindowState ) );
+
   return statusbar.init ();
 
 }
 
+function onDidChangeWindowState ( event ) {
+
+  if (event.focused && Utils.state.get ( 'didOpenInBrowser', false ) ) {
```

Comment:
This looks wrong, should we check if `didOpenInBrowser` is `true` instead?

Reply:
It is checking whether `didOpenInBrowser` is `true` 😳

(I have tested it using fake notifications)

Reply:
Right, I misread that, sorry 😬